README.rst

```diff
@@ -23,25 +23,30 @@ Who uses Sphinx-Gallery
 =======================
 
 * `Sphinx-Gallery <https://sphinx-gallery.github.io/stable/auto_examples/index.html>`_
-* `Scikit-learn <http://scikit-learn.org/dev/auto_examples/index.html>`_
-* `Nilearn <https://nilearn.github.io/auto_examples/index.html>`_
-* `MNE-python <https://www.martinos.org/mne/stable/auto_examples/index.html>`_
+
+.. projects_list_start
+
```

Comment:
Sorry to nit, does it work with this line removed?

Reply:
Sorry, but I'm not sure what you mean exactly: the label `.. projects_list_start` is required to indicate where the range of lines to include starts; it must be _below_ the entry for sphinx gallery because that is treated differently in README.rst and projects_list.rst (simple link vs :ref:).

Reply:
Sorry I mean no blank line between '.. projects_list_start' and the start of the bullet points like so:

```rst
.. projects_list_start
* `Scikit-learn <http://scikit-learn.org/stable/auto_examples/index.html>`_
...
```

Reply:
Oh, sorry - my bad, I didn't understand that you were referring to the empty lines. I already tried this but even without any empty lines in-between like so:
```
* `Sphinx-Gallery <https://sphinx-gallery.github.io/stable/auto_examples/index.html>`_
.. projects_list_start
* `Scikit-learn <http://scikit-learn.org/stable/auto_examples/index.html>`_
```
Sphinx (or docutils better to say) starts a new bullet list. I didn't find a way to remove the space between the first line and the rest. That's why I decided to re-insert the empty lines around `.. projects_list_start` for better readability.

Reply:
Ah great, thanks for checking, these little things shouldn't annoy me so much :sweat_smile: